cobc/cobc.c

```diff
@@ -6838,526 +6856,329 @@ abort_if_too_many_continuation_lines (int pline_cnt, const char *filename, int l
 	}
 }
 
-static void
-make_new_continuation_line (const char *cfile_name, char *pline[CB_READ_AHEAD],
-			    int * const pline_cnt, int line_num)
-{
-	const int margin_a = cobc_get_margin_a (1);
-	const int indicator = cobc_get_indicator ();
-	const int sequence_col = cobc_get_text_column ();
-	abort_if_too_many_continuation_lines (*pline_cnt + 1, cfile_name,
-					      line_num);
-	if (pline[*pline_cnt + 1] == NULL) {
-		pline[*pline_cnt + 1] = cobc_malloc (CB_LINE_LENGTH + 2);
+/* 
+	strncpy with skip multiple space for the listing mode
+	in_copy = margin_a for the margin
+	if is new string line replace don't skip the spaces
+*/
+static int
+strncpy_listing (char *dst, char *src, int n, int first, int in_copy, int is_new)
+{
+	int	i = 0;
+	const int	margin_a = cobc_get_margin_a (1);
+	int	cnt_space = 0;
+	int	j = 0;
+
```

Comment:
Looks like most of `DEBUG_REPLACE` is now out, was this explicit planned?
Did you use that to get an idea where the issue is?

Reply:
ping @sbelondr